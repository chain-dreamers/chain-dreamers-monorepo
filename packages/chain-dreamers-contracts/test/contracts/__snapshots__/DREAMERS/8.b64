data:application/json;base64,ZGF0YTphcHBsaWNhdGlvbi9qc29uLHsiaW1hZ2VfZGF0YSI6ICIlM2NzdmclMjB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnJTIwdmlld0JveD0nMCUyMDAlMjAyNTUlMjAyNTUnJTIwd2lkdGg9JzUwMHB4JyUyMGhlaWdodD0nNTAwcHgnJTNlJTNjcGF0aCUyMGQ9J00wLDBIMjU1VjI1NUgwVjAnJTIwZmlsbD0nJTIzNzdEOTU5JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE3MywyMTdIODJDNTQsMjI0LDQ4LDI1NSw0OCwyNTVIMjA3QzIwNywyNTUsMjAxLDIyNCwxNzMsMjE3JyUyMGZpbGw9JyUyM0NFQ0Y2NCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MiwyMTFIMTczVjI1NUg4MlYyMTEnJTIwZmlsbD0nJTIzRkVGN0VBJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTgzLDIxMVYyMjVDOTYsMjM4LDExMSwyNDYsMTI3LDI0NkMxNDQsMjQ2LDE1OSwyMzgsMTcyLDIyNVYyMTFIODMnJTIwZmlsbD0nJTIzRTREN0MyJy8lM2UlM2NwYXRoJTIwZD0nTTE5NywxMDZINThDNDYsMTA2LDM2LDExNywzNiwxMzFDMzYsMTQ1LDQ2LDE1Niw1OCwxNTZIMTk3QzIwOSwxNTYsMjE5LDE0NSwyMTksMTMxQzIxOSwxMTcsMjA5LDEwNiwxOTcsMTA2JyUyMGZpbGw9JyUyM0FDQTc5RCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OCwxMzVBNjksOTksMCwxLDAsMTk3LDEzNUE2OSw5OSwwLDEsMCw1OCwxMzUnJTIwZmlsbD0nJTIzRkVGN0VBJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5NiwxMjBDMTk2LDEyMCwyMDksMTE4LDIxMCwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OSwxMjBDNTksMTIwLDQ2LDExOCw0NSwxMzUnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J003OSwxMzVDNzksODYsMTA0LDQ1LDEzNiwzOEE0NCw0NCwwLDAsMCwxMjYsMzdDODksMzcsNTksODEsNTksMTM1QzU5LDE5MCw4OSwyMzMsMTI2LDIzM0E1MCw1MCwwLDAsMCwxMzYsMjMyQzEwNCwyMjUsNzksMTg1LDc5LDEzNSclMjBmaWxsPSclMjNFNEQ3QzInLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE5NEM5NywxOTQsOTAsMTg0LDkwLDE4NEM5MCwxODQsMTAxLDIxNywxMjcsMjE3QzE1NCwyMTcsMTY1LDE4NCwxNjUsMTg0QzE2NSwxODQsMTU4LDE5NCwxMjcsMTk0JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMjcsMTk0Qzk3LDE5NCw5MCwxODQsOTAsMTg0QTY0LDY0LDAsMCwwLDk0LDE5M0M5OSwxOTYsMTA5LDIwMCwxMjcsMjAwQzE0NiwyMDAsMTU2LDE5NiwxNjEsMTkzQTYwLDYwLDAsMCwwLDE2NSwxODRDMTY1LDE4NCwxNTgsMTk0LDEyNywxOTQnJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk5LDIwMUMxMDYsMjA5LDExNSwyMTcsMTI3LDIxN0MxNDAsMjE3LDE0OSwyMDksMTU2LDIwMUE1OSw1OSwwLDAsMSw5OSwyMDFWMjAxJyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMzUsMjA5QTQ5LDQ5LDAsMCwwLDE1MCwyMDdBMzMsMzMsMCwwLDAsMTUzLDIwM0E0Nyw0NywwLDAsMSwxMzMsMjA5SDEzNSclMjBmaWxsPSclMjNDN0M3QzcnLyUzZSUzY3BhdGglMjBkPSdNMTAyLDIwM0MxMDcsMjA5LDExNCwyMTQsMTIzLDIxNEgxMjVBMjQsMjQsMCwwLDEsMTEwLDIwN0EzOSwzOSwwLDAsMSwxMDIsMjAzJyUyMGZpbGw9JyUyM0M3QzdDNycvJTNlJTNjcGF0aCUyMGQ9J005NywxODlDOTMsMTg4LDkxLDE4Niw5MSwxODZBNDIsNDIsMCwwLDAsOTUsMTkzQTI5LDI5LDAsMCwwLDEwMiwxOTZMOTksMTk1QTQ2LDQ2LDAsMCwxLDk3LDE4OSclMjBmaWxsPSclMjNDN0M3QzcnLyUzZSUzY3BhdGglMjBkPSdNMTM0LDE5NEE4OSw4OSwwLDAsMSwxMTQsMTk0QzExOSwxOTUsMTI3LDE5NiwxMzksMTk1QzE0OSwxOTUsMTU2LDE5MywxNjEsMTkxQTQ0LDQ0LDAsMCwwLDE2NCwxODZDMTY0LDE4NiwxNTgsMTkzLDEzNCwxOTQnJTIwZmlsbD0nJTIzQzdDN0M3Jy8lM2UlM2NwYXRoJTIwZD0nTTExMiwxNThWMTY1QTE0LDE0LDAsMCwwLDExNywxNzVDMTE5LDE3NywxMjIsMTc5LDEyNywxNzlDMTMyLDE3OSwxMzYsMTc3LDEzOCwxNzVBMTQsMTQsMCwwLDAsMTQzLDE2NVYxNTgnJTIwZmlsbD0nbm9uZSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MywxMTdDOTgsMTEzLDEwNywxMjEsMTExLDEyNUE2LDYsMCwxLDAsMTIxLDExN0MxMDEsOTYsODMsMTE3LDgzLDExNyclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTM0LDExN0E2LDYsMCwwLDAsMTM4LDEyOEE2LDYsMCwwLDAsMTQ0LDEyNUMxNDgsMTIxLDE1NywxMTMsMTcyLDExN0MxNzIsMTE3LDE1NCw5NiwxMzQsMTE3JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMDYsMTIyQTE0LDE0LDAsMCwwLDEwMSwxMjJBMTUsMTUsMCwwLDAsMTAxLDE1MkExNCwxNCwwLDAsMCwxMDYsMTUxQTE1LDE1LDAsMCwxLDEwNiwxMjInJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE0OCwxMzdBMTUsMTUsMCwwLDEsMTU4LDEyMkExNCwxNCwwLDAsMCwxNTQsMTIyQTE1LDE1LDAsMCwwLDE1NCwxNTJBMTQsMTQsMCwwLDAsMTU4LDE1MUExNSwxNSwwLDAsMSwxNDgsMTM3JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J005NiwxMzdBMTAsMTQsMCwxLDAsMTE3LDEzN0ExMCwxNCwwLDEsMCw5NiwxMzcnJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE0OCwxMzdBMTAsMTQsMCwxLDAsMTY5LDEzN0ExMCwxNCwwLDEsMCwxNDgsMTM3JyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMDYsMTI2QTUsNSwwLDEsMCwxMTEsMTMxQTUsNSwwLDAsMCwxMDYsMTI2JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNTgsMTI2QTUsNSwwLDEsMCwxNjQsMTMxQTUsNSwwLDAsMCwxNTgsMTI2JyUyMGZpbGw9JyUyMzAwMDAwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J005OCw4N0MxMTIsODcsMTA5LDk4LDEyNCw5NUMxNDAsOTMsMTQ1LDgxLDE1Nyw5MSclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTExMCwxMDFDMTE4LDEwMSwxMTYsMTA3LDEyNiwxMDZDMTM1LDEwNCwxMzgsOTcsMTQ1LDEwMyclMjBmaWxsPSdub25lJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTU5LDE1NkM1OSwxNTYsNTIsMTgyLDcwLDE5OUM3OSwyMDYsODEsMjE4LDgxLDIxOEM4MSwyMTgsNzQsMTMyLDU5LDE1NiclMjBmaWxsPSclMjM3QUNCRTEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTk2LDE1NkMxODEsMTMyLDE3NCwyMTgsMTc0LDIxOEMxNzQsMjE4LDE3NiwyMDYsMTg1LDE5OUMyMDMsMTgyLDE5NiwxNTYsMTk2LDE1NiclMjBmaWxsPSclMjM3QUNCRTEnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNjEsMTA2VjEwNkExNSwxNSwwLDAsMCw1OCwxMDZDNDYsMTA2LDM2LDExNywzNiwxMzFDMzYsMTQ1LDQ2LDE1Niw1OCwxNTZINTlDNTksMTU2LDY5LDE1MCw2NSwxMzFDNjAsMTEyLDYxLDEwNiw2MSwxMDYnJTIwZmlsbD0nJTIzNjk2QTZBJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5NywxMDZBMTQsMTQsMCwwLDAsMTk0LDEwNlYxMDZDMTk0LDEwNiwxOTUsMTEyLDE5MCwxMzFDMTg2LDE1MCwxOTYsMTU2LDE5NiwxNTZIMTk3QzIwOSwxNTYsMjE5LDE0NSwyMTksMTMxQzIxOSwxMTcsMjA5LDEwNiwxOTcsMTA2JyUyMGZpbGw9JyUyMzY5NkE2QSclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J002MCwxMTNBMTgsMTgsMCwwLDEsNjAsMTA4VjEwOEM0NywxMDcsMzcsMTE4LDM3LDEzMUMzNywxNDMsNDUsMTUzLDU1LDE1NEEyNCwyNCwwLDAsMSw0NiwxMzVDNDYsMTI1LDUyLDExNiw2MCwxMTMnJTIwZmlsbD0nJTIzNEE0QjRCJy8lM2UlM2NwYXRoJTIwZD0nTTE5NSwxMDhWMTA4QTI0LDI0LDAsMCwxLDE5NSwxMTNDMjAzLDExNiwyMDksMTI1LDIwOSwxMzVBMjMsMjMsMCwwLDEsMjAwLDE1NEMyMTAsMTUzLDIxOCwxNDMsMjE4LDEzMUMyMTgsMTE4LDIwOCwxMDcsMTk1LDEwOCclMjBmaWxsPSclMjM0QTRCNEInLyUzZSUzY3BhdGglMjBkPSdNNjgsMTU1QzY2LDE1Myw2NSwxNTMsNjQsMTUzQzYzLDE1NSw2MCwxNTcsNjAsMTU3QzUzLDE4OSw3MiwxODgsODAsMjEzTDc4LDE5N0M2MywxODIsNjQsMTY0LDY4LDE1NSclMjBmaWxsPSclMjMzQUEyQzcnLyUzZSUzY3BhdGglMjBkPSdNMTk1LDE1N1YxNTdDMTk1LDE1NywxOTIsMTU1LDE5MSwxNTNDMTkwLDE1MywxODksMTUzLDE4NywxNTVDMTkxLDE2NCwxOTIsMTgyLDE3NywxOTdMMTc1LDIxM0MxODMsMTg4LDIwMiwxOTAsMTk1LDE1NyclMjBmaWxsPSclMjMzQUEyQzcnLyUzZSUzY3BhdGglMjBkPSdNMTExLDY0QzEwOSw2NCwxMDcsNjQsMTA2LDY2QTUsNSwwLDAsMSw5OCw2NkM5Nyw2NCw5NSw2NCw5Miw2NEM5MCw2NCw4Niw2NSw4NSw2N0M4NCw2OSw4NCw3MCw4NSw3M0M4Nyw4MCw5Miw4Niw5Niw5MEE2Nyw2NywwLDAsMCwxMDIsOTRDMTA3LDkxLDExNiw4MywxMTksNzNDMTIwLDcwLDEyMCw2OCwxMTksNjdDMTE4LDY1LDExNSw2NCwxMTEsNjQnJTIwZmlsbD0nJTIzOTQ2QTY5JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE3MCw2N0MxNjksNjUsMTY2LDY0LDE2Myw2NEMxNTksNjQsMTU4LDY0LDE1Nyw2NkE1LDUsMCwwLDEsMTQ5LDY2QzE0OCw2NCwxNDYsNjQsMTQ0LDY0QzE0MSw2NCwxMzcsNjUsMTM2LDY3QzEzNSw2OSwxMzUsNzAsMTM2LDczQzEzOSw4MywxNDgsOTEsMTUzLDk0QzE1NSw5MywxNTcsOTIsMTU5LDkwQzE2Myw4NiwxNjgsODAsMTcwLDczUTE3Miw2OSwxNzAsNjcnJTIwZmlsbD0nJTIzOTQ2QTY5JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE4NSw2NFY2NEEzLDMsMCwwLDAsMTgyLDY2QzE4MCw2OSwxNzYsNjcsMTc2LDY3VjY3QzE3NCw1OCwxNTksNTYsMTUzLDYyQzE0OCw1NiwxMzMsNTgsMTMxLDY3SDEyNEMxMjIsNTgsMTA3LDU2LDEwMiw2MkM5Niw1Niw4MSw1OCw3OSw2N1Y2N0M3OSw2Nyw3NSw2OSw3Myw2NkEzLDMsMCwwLDAsNzAsNjRWNjRBMywzLDAsMCwwLDY3LDY3QzY3LDcxLDcwLDc3LDgxLDc3Vjc3Qzg0LDgzLDg5LDg5LDkzLDkzQTY3LDY3LDAsMCwwLDEwMiwxMDFDMTAyLDEwMSwxMTgsOTAsMTI0LDc0QTIsMiwwLDAsMCwxMjQsNzRIMTMxVjc0QzEzNiw5MCwxNTMsMTAxLDE1MywxMDFBNzEsNzEsMCwwLDAsMTYyLDkzQTU1LDU1LDAsMCwwLDE3NCw3N1Y3N0MxODUsNzcsMTg4LDcxLDE4OCw2N0EzLDMsMCwwLDAsMTg1LDY0TTExOSw3M0MxMTYsODMsMTA3LDkxLDEwMiw5NEE2Nyw2NywwLDAsMSw5Niw5MEM5Miw4Niw4Nyw4MCw4NSw3M1E4Myw2OSw4NSw2N0M4Niw2NSw4OSw2NCw5Miw2NEM5Niw2NCw5Nyw2NCw5OCw2NkE1LDUsMCwwLDAsMTA2LDY2QzEwNyw2NCwxMDksNjQsMTExLDY0QzExNCw2NCwxMTgsNjUsMTE5LDY3QzEyMCw2OSwxMjAsNzAsMTE5LDczTTE3MCw3M0MxNjgsODAsMTYzLDg2LDE1OSw5MEMxNTcsOTIsMTU1LDkzLDE1Myw5NEMxNDgsOTEsMTM5LDgzLDEzNiw3M1ExMzUsNjksMTM2LDY3QzEzNyw2NSwxNDAsNjQsMTQ0LDY0QzE0Nyw2NCwxNDgsNjQsMTQ5LDY2QTUsNSwwLDAsMCwxNTcsNjZDMTU4LDY0LDE2MCw2NCwxNjMsNjRDMTY1LDY0LDE2OSw2NSwxNzAsNjdDMTcxLDY5LDE3MSw3MCwxNzAsNzMnJTIwZmlsbD0nJTIzQkNBMDRBJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEwNiw2NkE1LDUsMCwwLDEsMTA0LDY3TDEwNiw2OEE0LDQsMCwwLDAsMTA5LDY2QTcsNywwLDAsMSwxMTIsNjRIMTExQzEwOSw2NCwxMDcsNjUsMTA2LDY2JyUyMGZpbGw9JyUyMzQ3MjkyOScvJTNlJTNjcGF0aCUyMGQ9J004OCw3M0M4Nyw3MSw4Nyw2OSw4OCw2N0E4LDgsMCwwLDEsOTMsNjRIOTJDODksNjQsODcsNjUsODUsNjdDODQsNjksODQsNzAsODUsNzJDODgsNzksOTMsODUsOTcsODlDOTgsOTEsMTAwLDkyLDEwMiw5NEwxMDMsOTNMMTAwLDkxQzk2LDg3LDkwLDgxLDg4LDczJyUyMGZpbGw9JyUyMzQ3MjkyOScvJTNlJTNjcGF0aCUyMGQ9J00xNDksNjZDMTQ4LDY1LDE0Niw2NCwxNDQsNjRIMTQzQTYsNiwwLDAsMSwxNDYsNjZBNCw0LDAsMCwwLDE0OSw2OEwxNTEsNjdBNSw1LDAsMCwxLDE0OSw2NiclMjBmaWxsPSclMjM0NzI5MjknLyUzZSUzY3BhdGglMjBkPSdNMTcwLDY3QzE2OCw2NSwxNjYsNjQsMTYzLDY0SDE2MkE4LDgsMCwwLDEsMTY3LDY3QzE2OCw2OSwxNjgsNzEsMTY3LDczQzE2NSw4MSwxNTksODcsMTU1LDkxTDE1Miw5M0wxNTMsOTRDMTU1LDkyLDE1Nyw5MSwxNTgsODlDMTYyLDg1LDE2Nyw3OSwxNzAsNzJDMTcxLDcwLDE3MSw2OCwxNzAsNjcnJTIwZmlsbD0nJTIzNDcyOTI5Jy8lM2UlM2NzdHlsZSUzZXBhdGh7c3Ryb2tlLXdpZHRoOjAuNzF9JTNjL3N0eWxlJTNlJTNjL3N2ZyUzZSIsICJuYW1lIjogIkRyZWFtZXIlMjAlMjM4IiwgImRlc2NyaXB0aW9uIjogIlJ1bm5lcnMlMjBydW4sJTIwYnV0JTIwc29tZXRpbWVzJTIwdGhleSUyMGRyZWFtLiUyMFRoaXMlMjBpcyUyMG9uZSUyMG9mJTIwdGhlaXIlMjBkcmVhbXMuIiwiYXR0cmlidXRlcyI6ICIifQ==