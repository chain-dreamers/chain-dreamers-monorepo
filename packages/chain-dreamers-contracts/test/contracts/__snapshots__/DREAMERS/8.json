data:application/json,{"image_data": "%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0,0H255V255H0V0'%20fill='%2377D959'%20stroke='%23000'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%23CECF64'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23FEF7EA'%20stroke='%23000'/%3e%3cpath%20d='M83,211V225C96,238,111,246,127,246C144,246,159,238,172,225V211H83'%20fill='%23E4D7C2'/%3e%3cpath%20d='M197,106H58C46,106,36,117,36,131C36,145,46,156,58,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23ACA79D'%20stroke='%23000'/%3e%3cpath%20d='M58,135A69,99,0,1,0,197,135A69,99,0,1,0,58,135'%20fill='%23FEF7EA'%20stroke='%23000'/%3e%3cpath%20d='M196,120C196,120,209,118,210,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,120C59,120,46,118,45,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M79,135C79,86,104,45,136,38A44,44,0,0,0,126,37C89,37,59,81,59,135C59,190,89,233,126,233A50,50,0,0,0,136,232C104,225,79,185,79,135'%20fill='%23E4D7C2'/%3e%3cpath%20d='M127,194C97,194,90,184,90,184C90,184,101,217,127,217C154,217,165,184,165,184C165,184,158,194,127,194'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M127,194C97,194,90,184,90,184A64,64,0,0,0,94,193C99,196,109,200,127,200C146,200,156,196,161,193A60,60,0,0,0,165,184C165,184,158,194,127,194'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M99,201C106,209,115,217,127,217C140,217,149,209,156,201A59,59,0,0,1,99,201V201'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M135,209A49,49,0,0,0,150,207A33,33,0,0,0,153,203A47,47,0,0,1,133,209H135'%20fill='%23C7C7C7'/%3e%3cpath%20d='M102,203C107,209,114,214,123,214H125A24,24,0,0,1,110,207A39,39,0,0,1,102,203'%20fill='%23C7C7C7'/%3e%3cpath%20d='M97,189C93,188,91,186,91,186A42,42,0,0,0,95,193A29,29,0,0,0,102,196L99,195A46,46,0,0,1,97,189'%20fill='%23C7C7C7'/%3e%3cpath%20d='M134,194A89,89,0,0,1,114,194C119,195,127,196,139,195C149,195,156,193,161,191A44,44,0,0,0,164,186C164,186,158,193,134,194'%20fill='%23C7C7C7'/%3e%3cpath%20d='M112,158V165A14,14,0,0,0,117,175C119,177,122,179,127,179C132,179,136,177,138,175A14,14,0,0,0,143,165V158'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M83,117C98,113,107,121,111,125A6,6,0,1,0,121,117C101,96,83,117,83,117'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M134,117A6,6,0,0,0,138,128A6,6,0,0,0,144,125C148,121,157,113,172,117C172,117,154,96,134,117'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M106,122A14,14,0,0,0,101,122A15,15,0,0,0,101,152A14,14,0,0,0,106,151A15,15,0,0,1,106,122'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M148,137A15,15,0,0,1,158,122A14,14,0,0,0,154,122A15,15,0,0,0,154,152A14,14,0,0,0,158,151A15,15,0,0,1,148,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M96,137A10,14,0,1,0,117,137A10,14,0,1,0,96,137'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M148,137A10,14,0,1,0,169,137A10,14,0,1,0,148,137'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M106,126A5,5,0,1,0,111,131A5,5,0,0,0,106,126'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M158,126A5,5,0,1,0,164,131A5,5,0,0,0,158,126'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M98,87C112,87,109,98,124,95C140,93,145,81,157,91'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M110,101C118,101,116,107,126,106C135,104,138,97,145,103'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,156C59,156,52,182,70,199C79,206,81,218,81,218C81,218,74,132,59,156'%20fill='%237ACBE1'%20stroke='%23000'/%3e%3cpath%20d='M196,156C181,132,174,218,174,218C174,218,176,206,185,199C203,182,196,156,196,156'%20fill='%237ACBE1'%20stroke='%23000'/%3e%3cpath%20d='M61,106V106A15,15,0,0,0,58,106C46,106,36,117,36,131C36,145,46,156,58,156H59C59,156,69,150,65,131C60,112,61,106,61,106'%20fill='%23696A6A'%20stroke='%23000'/%3e%3cpath%20d='M197,106A14,14,0,0,0,194,106V106C194,106,195,112,190,131C186,150,196,156,196,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23696A6A'%20stroke='%23000'/%3e%3cpath%20d='M60,113A18,18,0,0,1,60,108V108C47,107,37,118,37,131C37,143,45,153,55,154A24,24,0,0,1,46,135C46,125,52,116,60,113'%20fill='%234A4B4B'/%3e%3cpath%20d='M195,108V108A24,24,0,0,1,195,113C203,116,209,125,209,135A23,23,0,0,1,200,154C210,153,218,143,218,131C218,118,208,107,195,108'%20fill='%234A4B4B'/%3e%3cpath%20d='M68,155C66,153,65,153,64,153C63,155,60,157,60,157C53,189,72,188,80,213L78,197C63,182,64,164,68,155'%20fill='%233AA2C7'/%3e%3cpath%20d='M195,157V157C195,157,192,155,191,153C190,153,189,153,187,155C191,164,192,182,177,197L175,213C183,188,202,190,195,157'%20fill='%233AA2C7'/%3e%3cpath%20d='M111,64C109,64,107,64,106,66A5,5,0,0,1,98,66C97,64,95,64,92,64C90,64,86,65,85,67C84,69,84,70,85,73C87,80,92,86,96,90A67,67,0,0,0,102,94C107,91,116,83,119,73C120,70,120,68,119,67C118,65,115,64,111,64'%20fill='%23946A69'%20stroke='%23000'/%3e%3cpath%20d='M170,67C169,65,166,64,163,64C159,64,158,64,157,66A5,5,0,0,1,149,66C148,64,146,64,144,64C141,64,137,65,136,67C135,69,135,70,136,73C139,83,148,91,153,94C155,93,157,92,159,90C163,86,168,80,170,73Q172,69,170,67'%20fill='%23946A69'%20stroke='%23000'/%3e%3cpath%20d='M185,64V64A3,3,0,0,0,182,66C180,69,176,67,176,67V67C174,58,159,56,153,62C148,56,133,58,131,67H124C122,58,107,56,102,62C96,56,81,58,79,67V67C79,67,75,69,73,66A3,3,0,0,0,70,64V64A3,3,0,0,0,67,67C67,71,70,77,81,77V77C84,83,89,89,93,93A67,67,0,0,0,102,101C102,101,118,90,124,74A2,2,0,0,0,124,74H131V74C136,90,153,101,153,101A71,71,0,0,0,162,93A55,55,0,0,0,174,77V77C185,77,188,71,188,67A3,3,0,0,0,185,64M119,73C116,83,107,91,102,94A67,67,0,0,1,96,90C92,86,87,80,85,73Q83,69,85,67C86,65,89,64,92,64C96,64,97,64,98,66A5,5,0,0,0,106,66C107,64,109,64,111,64C114,64,118,65,119,67C120,69,120,70,119,73M170,73C168,80,163,86,159,90C157,92,155,93,153,94C148,91,139,83,136,73Q135,69,136,67C137,65,140,64,144,64C147,64,148,64,149,66A5,5,0,0,0,157,66C158,64,160,64,163,64C165,64,169,65,170,67C171,69,171,70,170,73'%20fill='%23BCA04A'%20stroke='%23000'/%3e%3cpath%20d='M106,66A5,5,0,0,1,104,67L106,68A4,4,0,0,0,109,66A7,7,0,0,1,112,64H111C109,64,107,65,106,66'%20fill='%23472929'/%3e%3cpath%20d='M88,73C87,71,87,69,88,67A8,8,0,0,1,93,64H92C89,64,87,65,85,67C84,69,84,70,85,72C88,79,93,85,97,89C98,91,100,92,102,94L103,93L100,91C96,87,90,81,88,73'%20fill='%23472929'/%3e%3cpath%20d='M149,66C148,65,146,64,144,64H143A6,6,0,0,1,146,66A4,4,0,0,0,149,68L151,67A5,5,0,0,1,149,66'%20fill='%23472929'/%3e%3cpath%20d='M170,67C168,65,166,64,163,64H162A8,8,0,0,1,167,67C168,69,168,71,167,73C165,81,159,87,155,91L152,93L153,94C155,92,157,91,158,89C162,85,167,79,170,72C171,70,171,68,170,67'%20fill='%23472929'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e", "name": "Dreamer%20%238", "description": "Runners%20run,%20but%20sometimes%20they%20dream.%20This%20is%20one%20of%20their%20dreams.", "attributes": ""}
